doc/development/developing-sql-dialect/developing_a_dialect.md

```diff
@@ -89,7 +89,7 @@ The Java package structure of the `virtualschema-jdbc-adapter` reflects the sepa
       |     |
       |     '-- ...
       |
-      '-- jdbc                                 Base implementation for getting metadata from JDBC
+      '-- jdbc                                 Base implementation for getting metadata from JDBC (based in virtual-schema-common-jdbc repository) 
```

Comment:
would be nice to add the links for each repository when they are mentioned, as for example:

`Base implementation for getting metadata from JDBC (based in [virtual-schema-common-jdbc repository](https://github.com/exasol/virtual-schema-common-jdbc)) `

Reply:
Links don't work inside of this schema, I've tried it

Reply:
I've listed repositories and added links there, outside of the schemas